### PR TITLE
[ENG-4599] Tim/add send sip10 and reorganise

### PR DESCRIPTION
**Pull request by teebszet**
context: https://linear.app/xverseapp/issue/ENG-4599/look-into-post-conditions-that-were-constructing-these-properly-tim

changes: 
- added send sip10 route
- refactored to split into routes and use mantine UI

### CODE REVIEW

**Comment by teebszet** [comment]
@aryzing I'm not sure if I have the encoding right here to send 100LEO to an address. any ideas?

I get this error on the popup: 
image

File: example/src/components/stacks/SendSip10/index.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        contract: form.contract,
+        functionName: 'transfer',
+        arguments: [
+          uintCV(+form.amount),
+          standardPrincipalCV(stxAddressInfo?.[0].address),
+          standardPrincipalCV(form.address),
+          noneCV(),
+          //form.memo ? someCV(bufferCV(Buffer.from(form.memo))) : noneCV(),
+        ].map((arg) => cvToString(arg)),
+      });
```
Comment (teebszet) [inline example/src/components/stacks/SendSip10/index.tsx:79]: @m-aboelenein @aryzing any ideas on whether there's something causing the error here?

Comment (aryzing) [reply example/src/components/stacks/SendSip10/index.tsx:79]: In the map, try with `cvToHex()`

**Review by terencehh — APPROVED** [review]
tested working

**Review by JanKortes212 — COMMENTED** [review]
mqe1

File: example/src/components/stacks/SendSip10/index.tsx (hunk 1 above)
Comment (teebszet) [reply example/src/components/stacks/SendSip10/index.tsx:79]: nice! thank you, i'll try that and make a note to update the docs https://docs.xverse.app/sats-connect/stacks-methods/stx_callcontract

actually a lot of the stx docs seem to be incomplete

**Pull request merged**